bin/rolling_index

```diff
@@ -20,7 +20,7 @@ Daemons.run_proc('rolling index') do
         # Clean up cruft in QA and stage
         Indexer.delete(solr: solr_conn, identifier: identifier) if ['stage', 'qa'].include?(ENV['HONEYBADGER_ENV'])
       rescue Dor::Services::Client::UnexpectedResponse
-        Honeybadger.notify('Unexpected response from Dor Services App for ', { druid: identifier })
+        Honeybadger.notify("Unexpected response from Dor Services App for #{identifier}.")
```

Comment:
Why do you want this in the message rather than in the context?  Typically the message should be more generic.

Reply:
I was misunderstanding how the context works. Have updated.